### PR TITLE
Remove Perl dependency

### LINKED ISSUE #4: Remove Perl dependency
These should be no `perl` dependency. It can be rewritten to use bash tools like `sed` or `php` instead.

### PR DESCRIPTION
**Pull request by lukecarbis**
Resolves #4.

### CODE REVIEW

File: init-plugin.sh [hunk 1]
```diff
@@ -16,15 +16,16 @@ if [ -z "$name" ]; then
...
 fi
 
-slug=$( perl -pe '$_ = lc; s/ /-/g' <<< "$name" )
-prefix=$( perl -pe '$_ = lc; s/ /_/g' <<< "$name" )
-namespace=$( perl -pe 's/ //g' <<< "$name" )
-class=$( perl -pe 's/ /_/g' <<< "$name" )
+slug="$( echo "$name" | tr '[:upper:]' '[:lower:]' | sed 's/ /-/' )"
+prefix="$( echo "$name" | tr '[:upper:]' '[:lower:]' | sed 's/ /_/' )"
+namespace="$( echo "$name" | sed 's/ //' )"
+class="$( echo "$name" | sed 's/ /_/' )"
```
Comment (westonruter) [inline init-plugin.sh]: The above `sed` calls are missing the `g` modifier for global replacements, I believe.

**Comment by lukecarbis** [comment]
@westonruter Fixed.

File: init-plugin.sh [hunk 2]
```diff
@@ -65,11 +66,12 @@ cd tests
...
-perl -p -i'' -e "s/Foo Bar/$name/g" $( find */ -type f ) *.*
-perl -p -i'' -e "s/foo-bar/$slug/g" $( find */ -type f  ) *.*
-perl -p -i'' -e "s/foo_bar/$prefix/g" $( find */ -type f ) *.*
-perl -p -i'' -e "s/FooBar/$namespace/g" $( find */ -type f ) *.*
-perl -p -i'' -e "s/Foo_Bar/$class/g" $( find */ -type f ) *.*
+git grep -lz "Foo Bar" | xargs -0 sed -i '' -e "s/Foo Bar/$name/g"
+git grep -lz "foo-bar" | xargs -0 sed -i '' -e "s/foo-bar/$slug/g"
+git grep -lz "foo_bar" | xargs -0 sed -i '' -e "s/foo_bar/$prefix/g"
+git grep -lz "FooBar" | xargs -0 sed -i '' -e "s/FooBar/$namespace/g"
+git grep -lz "Foo_Bar" | xargs -0 sed -i '' -e "s/Foo_Bar/$class/g"
```
Comment (behzod) [inline init-plugin.sh]: @westonruter and @lukecarbis please note that these sed commands with `-i ''` do not work on Linux.

Comment (westonruter) [reply init-plugin.sh]: Oh yeah! I remember now that's why I used `perl` to begin with.

**Pull request merged**